store/postgres/src/store.rs

```diff
@@ -233,6 +233,11 @@ impl BasicStore for Store {
                 .offset(range.skip as i64);
         }
 
+        {
+            debug!(self.logger, "find";
+                "sql" => format!("{:?}", debug_query::<Pg, _>(&diesel_query)));
```

Comment:
The alignment here is off by a few spaces. I'm also not sure why we need a dedicated scope around the `debug!` statement.